my-app/yarn.lock

```diff
@@ -841,13 +841,20 @@
     "@babel/helper-plugin-utils" "^7.8.3"
```

Comment:
Why do you have both

* `package-lock.json`
* `yarn.lock`

Please use only one package manager (yarn or npm) - thus leave one `.lock` file, and delete other.
IMO - it wouldn't make significant change which one you'll choose in terms of this project. But please stick with one 😃 

For more info https://blog.soshace.com/yarn-package-manager-in-2019-should-we-keep-on-comparing-yarn-with-npm/